docs/FAQ.md

```diff
@@ -247,6 +247,35 @@ the I/O and starting the initial processing can suffer.  But if you have a lot o
 cannot be done on the GPU, like complex UDFs, the more tasks you have the more CPU processing you
 can throw at it.
 
+### Why are multiple GPUs per executor not supported?
+
+The RAPIDS Accelerator only supports a single GPU per executor because that was a limitation of
+[RAPIDS cudf](https://github.com/rapidsai/cudf), the foundation of the Accelerator.  Basic support
+for working with multiple GPUs has only recently been added to RAPIDS cudf, and there are no plans
```

Comment:
should we link to the cudf issue?

Reply:
There are multiple issues, which one do I pick?